test/Data.purs

```diff
@@ -233,6 +214,13 @@ suite = do
             /\ "c25102f050fe938943acc45f65567fffffffff"
         ]
 
+-- | Concatenate two QuickCheck tests
+andResult :: Result -> Result -> Result
```

Comment:
Isn't it `Test.QuickCheck.Combinators.conj`?

Reply:
Thanks! I was looking for it!